Makefile

```diff
@@ -23,8 +23,14 @@ else
 endif
 
 REPOPATH ?= k8s.io/minikube
+K8S_VENDOR_PATH := $(REPOPATH)/vendor/k8s.io/kubernetes
 export GO15VENDOREXPERIMENT=1
 
+# Update this to match the vendored version of k8s in godeps.json
+K8S_VERSION := v1.3.0-alpha.3-838+ba170aa191f8c7
+# Set the version information in kubernetes.
+LD_FLAGS = "-s -w -X $(K8S_VENDOR_PATH)/pkg/version.gitVersion=$(K8S_VERSION) -X $(K8S_VENDOR_PATH)/pkg/version.gitCommit=$(shell git rev-parse HEAD)"
```

Comment:
Moved it to a Python script to parse the json. I added gitTreeState as well. There are two other arguments it looks like k8s adds, but the file says they're deprecated so I'm leaving them off for now: https://github.com/kubernetes/kubernetes/blob/master/pkg/version/base.go#L42